users/views.py

```diff
@@ -10,3 +21,68 @@
 logger.critical("Critical issue!")
 
 # Create your views here.
+
+User = get_user_model()
+
+
+class CustomPasswordResetView(APIView):
+    """
+    Request for password reset.
+    Sends an email if the user exists and returns clear feedback.
+    """
+    def post(self, request, *args, **kwargs):
+        email = request.data.get("email")
+        if not email:
+            return Response({"email": "This field is required."}, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
Consider using DRF's serializer validation for field presence (for future extensibility).